packages/api/src/auth/github/github.service.ts

```diff
@@ -1,19 +1,23 @@
 import { Injectable } from '@nestjs/common';
+import { UserModel } from 'src/users/user.model';
 
-import { User } from '@coderscamp/shared/models/user';
+import { RegisteredUser } from '@coderscamp/shared/models/user';
 
-import { UsersRepository } from '../../users/users.repository';
-import { UserFromGithub } from '../../users/users.types';
+import { GithubRepository } from './github.repository';
 
 @Injectable()
 export class GithubService {
-  constructor(private readonly usersRepository: UsersRepository) {}
+  constructor(private readonly githubRepository: GithubRepository) {}
 
-  async authorizeUser(githubUserData: UserFromGithub): Promise<User> {
-    let user = await this.usersRepository.getByGithubId(githubUserData.githubId);
+  async authorizeUser(githubUserData: Omit<RegisteredUser, 'id'>): Promise<RegisteredUser> {
```

Comment:
Once more: 
Omit<RegisteredUser, 'id'> 

looks like "Registered but not exactly, not already because do not have id" :P